src/components/ProductCard.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import '../styles/components/ProductCard.css';
+import PropTypes from 'prop-types';
+
+class MovieCard extends Component {
+  render() {
+    const { product } = this.props;
+    const { title, thumbnail, price } = product;
+    return (
+      <div className="movie-card" data-testid="product">
+        <img src={ thumbnail } alt={ `${title}` } />
+        <p>{ title }</p>
```

Comment:
E ai belezinha!? Queria dar uma dica, o que acha de colocarmos o title e price dentro de um container afim de podermos customiza-los melhor quando for preciso?``
```suggestion
     <div>
        <p>{ title }</p>
```